saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/registration/RelyingPartyRegistration.java

```diff
@@ -438,10 +441,12 @@ private static Saml2X509Credential fromDeprecated(
 
 		private final Saml2MessageBinding singleSignOnServiceBinding;
 
+		private List<String> signingAlgorithmUris;
```

Comment:
This should be a `Collection` since the `List`-specific methods are not needed.

Reply:
List specific are not needed, but the order matters, it the main reason I used list there

Reply:
I see, fair point. Let's leave it as-is.